nucypher/blockchain/eth/domains.py

```diff
@@ -0,0 +1,70 @@
+from enum import Enum
+from typing import NamedTuple
+
+
+class ChainInfo(NamedTuple):
+    id: int
+    name: str
+
+
+class EthChain(ChainInfo, Enum):
+    MAINNET = (1, "mainnet")
+    GOERLI = (5, "goerli")
+    SEPOLIA = (11155111, "sepolia")
+    TESTERCHAIN = (131277322940537, "eth-tester")
+
+
+class PolygonChain(ChainInfo, Enum):
+    POLYGON = (137, "polygon")
+    MUMBAI = (80001, "mumbai")
+    TESTERCHAIN = (131277322940537, "eth-tester")
+
+
+class DomainInfo(NamedTuple):
+    name: str
+    eth_chain: EthChain
+    polygon_chain: PolygonChain
+
+    def __str__(self) -> str:
+        return self.name
+
+    def __bytes__(self) -> bytes:
+        return self.name.encode()
```

Comment:
If we're going to access `.name` directly, these two methods are not beneficial.

Reply:
✔️